provision.sh

```diff
@@ -62,6 +62,9 @@ echo "[PROVISIONER] Installing node.js"
 curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
+echo "[PROVISIONER] Installing yarn"
+npm install -g yarn
```

Comment:
Maybe you'll need `sudo`

[side note]:
I've tried [fixing npm permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions) once and ruined my box 😄

Reply:
you're right → https://semaphoreci.com/renderedtext/boxbox/branches/bm-install-yarn/builds/1

Thanks